docs/user-guide/Authentication.md

```diff
@@ -4,10 +4,40 @@ RESTler supports token-based and certificate based authentication.
 
 **Token based authentication**
 
-The user must provide a separate program to generate tokens, which implements the authentication method required by the API.  This will be invoked in a separate process by RESTler to obtain and regularly refresh tokens.  When invoked, this program must print metadata about the tokens on the first line, followed by each token and the required token header on a separate line for each application.  For example:
+
+The user has three options for providing token based authentication; Module, Location, and CMD.
```

Comment:
> token based authentication; Module, Location, and CMD. [](http://example.com/codeflow?start=40&length=55)

It might be helpful to say after this sentence 'For details on the format of these options, please see SettingsFile.md'

Reply:
agreed, adding this